src/Bicep.LangServer/Completions/BicepCompletionContextKind.cs

```diff
@@ -198,6 +199,11 @@ public enum BicepCompletionContextKind : ulong
         /// <summary>
         /// The current location is after the output type.
         /// </summary>
-        OutputTypeFollower = 1UL << 36
+        OutputTypeFollower = 1UL << 36,
+
+        /// <summary>
+        /// The current location can accept a symbolic reference to a resource.
+        /// </summary>
+        ResourceDependsOnSymbolicReference = 1UL << 37
```

Comment:
Also minor: I think something like `ExpectsResourceSymbolicReference` or `ResourceDependency` would be more consistent with the existing context kind names.